client/flags/flags.go

```diff
@@ -112,7 +111,7 @@ func AddTxFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(FlagGasPrices, "", "Gas prices in decimal format to determine the transaction fee (e.g. 0.1uatom)")
 	cmd.Flags().String(FlagNode, "tcp://localhost:26657", "<host>:<port> to tendermint rpc interface for this chain")
 	cmd.Flags().Bool(FlagUseLedger, false, "Use a connected Ledger device")
-	cmd.Flags().Float64(FlagGasAdjustment, DefaultGasAdjustment, "adjustment factor to be multiplied against the estimate returned by the tx simulation; if the gas limit is set manually this flag is ignored ")
+	cmd.Flags().Float64(FlagGasAdjustment, 0, "adjustment factor to be multiplied against the estimate returned by the tx simulation; if the gas limit is set manually this flag is ignored ")
```

Comment:
This should still be 1.0 IMO

Reply:
the client will now use the value in `client.toml` as the default, instead of what's defined here. setting this to `0` means there is no default for the flag.

Reply:
if i have it as 1.0 here, the CLI help text will say `(default: 1)`, while in fact the default value is taken from `client.toml` and may be a value that's not 1.0

Reply:
Yes, so again, `client.toml` should _not_ be defining any defaults. The file should be completely opt-in. So `""` is treated as an OK value which means it would read the default value from the CLI flags.